docs/toc.yml

```diff
@@ -220,6 +221,17 @@ items:
         displayName: seq,logging
         href: logging/seq-integration.md
 
+  - name: Custom integrations
```

Comment:
Put this under integrations? Or is that going to pollute the namespace

Reply:
We could move it there to see what it looks like.

Reply:
Too polluted.